src/main/java/mods/eln/Eln.java

```diff
@@ -3821,6 +3900,32 @@ private void registerDust(int id) {
             Data.addResource(element.newItemStack());
             addToOre("dustIron", element.newItemStack());
         }
+        {
+            subId = 3;
+            completId = subId + (id << 6);
+
+            name = TR_NAME(Type.NONE, "Lapis Dust");
+            element = new GenericItemUsingDamageDescriptorWithComment(name,// iconId,
+                // name,
+                new String[]{});
+            dustCopper = element;
```

Comment:
Why is everything in this part of the file called `dustCopper` when they are not `dustCopper`??? This isn't with just your code, but would be nice if it was fixed while you're at it.

Reply:
Seems to be programming shorthand. I'm not sure I want to touch that.